tests/test_draw.py

```diff
@@ -1010,6 +1010,10 @@ def test_shoot():
     # It's not exactly the same, since the rngs are realized in a different order.
     np.testing.assert_allclose(image3.array, image1.array, rtol=0.25)
 
+    # Test that shooting with 0.0 flux makes a zero-photons image.
+    image4 = (obj*0).drawImage(method='phot')
+    np.testing.assert_equal(image4.array, image4.array*0)
```

Comment:
This is fine, but slightly wasteful.  You could just compare it to a literal `0`.

Reply:
Oh, interesting.  I didn't realize it would do the broadcast.  Thanks.